src/theory/shared_terms_database.h

```diff
@@ -254,14 +251,16 @@ class SharedTermsDatabase : public context::ContextNotifyObj {
   /**
    * get equality engine
    */
-  theory::eq::EqualityEngine* getEqualityEngine() { return &d_equalityEngine; }
-
-protected:
+  theory::eq::EqualityEngine* getEqualityEngine();
 
+ protected:
   /**
    * This method gets called on backtracks from the context manager.
    */
  void contextNotifyPop() override { backtrack(); }
+
+ /** Equality engine */
+ theory::eq::EqualityEngine* d_equalityEngine;
```

Comment:
It looks like ClangFormat messed up here. Is this just a single space indentation?